src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PAY_FOR_DAY = 40;
+  const MIN_DAYS_FOR_MAX_OFF = 7;
+  const MAX_OFF = 50;
+  const MIN_DAYS_FOR_FIRST_OFF = 3;
+  const MIN_OFF = 20;
+
+  if (days < MIN_DAYS_FOR_FIRST_OFF) {
+    return days * PAY_FOR_DAY;
+  }
+
+  if (days >= MIN_DAYS_FOR_FIRST_OFF && days < MIN_DAYS_FOR_MAX_OFF) {
```

Comment:
Instead of checking if 'days' is greater than or equal to 'MIN_DAYS_FOR_FIRST_OFF' again, you can omit this check. Since the code already returned for the case where 'days' is less than 'MIN_DAYS_FOR_FIRST_OFF', reaching this point means 'days' is at least 'MIN_DAYS_FOR_FIRST_OFF'.